guava/src/main/java/org/jdbi/v3/guava/GuavaCollectors.java

```diff
@@ -54,7 +57,7 @@
     /**
      * @return a collector into {@code ImmutableSortedSet<T>}
      */
-    public static <T extends Comparable<T>> Collector<T, ImmutableSortedSet.Builder<T>, ImmutableSortedSet<T>> toImmutableSortedSet() {
+    public static <T extends Comparable<T>> Collector<T, ?, ImmutableSortedSet<T>> toImmutableSortedSet() {
```

Comment:
Unrelated, but for completeness we should include an additional method `toImmutableSortedSet(Comparator<T>)`